lib/execute-method-map.ts

```diff
@@ -61,7 +61,8 @@ export const executeMethodMap = {
   'mobile: tapWithNumberOfTaps': {
     command: 'mobileTapWithNumberOfTaps',
     params: {
-      required: ['elementId', 'numberOfTouches', 'numberOfTaps'],
+      required: ['numberOfTouches', 'numberOfTaps'],
```

Comment:
How about making `numberOfTouches` optional as well (default to `1` if not provided)?

Reply:
done